index.md

```diff
@@ -5,4 +5,83 @@ title: issuehunter
 
 # Introduction
 
-What a wonderful project!
+Issuehunter is a decentralized application that enables Open Source crowdfunding
+on the Ethereum blockchain.
+
+It is a new way to improve Open Source development lifecycle by making it
+transparent and deterministic.
+
+It makes it easier for supporters to funds projects they love and for Open
+Source contributors to collect rewards for their work.
+
+The dapp is composed by three main elements:
+
+* a set of smart contracts deployed on the Ethereum blockchain
+* a service that helps verifying software patches submitted
+* a web based directory to discover funded issues that are still waiting to be
+  resolved
+
+## A new way of contributing to Open Source
+
+There are many ways to contribute to Open Source projects.
+
+You could be involved in Open Source by simply reporting bugs or answering to
+questions made by other users. You could fix or translate documentation. You
+could solve bigger issues, like fixing bugs or keeping dependencies up to date.
+You could be deeply involved in a project as a core contributor and be able to
+work on new features or to update existing ones.
+
+If you use OSS, but you don't have enough time or knowledge to help, you could
+be able to support its development by making donations.
+
+Donating to OS authors, projects, or communities, is one of the most
+forward-looking actions OSS users can do.
+
+Issuehunter enables new way of collaborating to Open Source projects.
+
+It lowers barriers to get access to funds for authors and it helps focusing the
+development of a project in specific areas for funders.
+
+## How it works
+
+### Step 1: fund an issue
+
+Contributors add funds to any unresolved issue on GitHub. The act of adding
+funds to an open issue triggers a smart contract function call and a transaction
+of Ethereum between the funder and the contract that is controlling the issue's
```

Comment:
and a transaction on the Ethereum blockchain with a fund transfer between the funder and the contract...